src/components/SelectionListRadio/BaseSelectionListRadio.js

```diff
@@ -0,0 +1,378 @@
+     *
+     *     1. It ALWAYS includes a list header and a list footer, even if we don't provide/render those.
+     *     2. Each section includes a header, even if we don't provide/render one.
+     *
+     *     For example, given a list with two sections, two items in each section, no header, no footer, and no section headers, the flat array might look something like this:
+     *
+     *     [{header}, {sectionHeader}, {item}, {item}, {sectionHeader}, {item}, {item}, {footer}]
+     *
+     * @returns {Object}
+     */
+    const getItemLayout = (data, flatDataArrayIndex) => {
+        const targetItem = flattenedSections.itemLayouts[flatDataArrayIndex];
+
+        return {
+            length: targetItem.length,
+            offset: targetItem.offset,
+            index: flatDataArrayIndex,
+        };
+    };
+
+    const renderItem = ({item, index, section}) => {
+        const isFocused = focusedIndex === index + lodashGet(section, 'indexOffset', 0);
+
+        return (
+            <PressableWithFeedback
+                ref={pressableRef}
+                onPress={() => props.onSelectRow(item)}
+                accessibilityLabel={item.text}
+                accessibilityRole="button"
+                hoverDimmingValue={1}
+                hoverStyle={styles.hoveredComponentBG}
+                focusStyle={styles.hoveredComponentBG}
+            >
+                <View style={[styles.flex1, styles.justifyContentBetween, styles.sidebarLinkInner, styles.optionRow, isFocused && styles.sidebarLinkActive]}>
+                    <View style={[styles.flex1, styles.alignItemsStart]}>
+                        <Text style={[styles.optionDisplayName, isFocused ? styles.sidebarLinkActiveText : styles.sidebarLinkText, item.isSelected && styles.sidebarLinkTextBold]}>
+                            {item.text}
+                        </Text>
+
+                        {Boolean(item.alternateText) && (
+                            <Text style={[isFocused ? styles.sidebarLinkActiveText : styles.sidebarLinkText, styles.optionAlternateText, styles.textLabelSupporting]}>
+                                {item.alternateText}
+                            </Text>
+                        )}
+                    </View>
+
+                    {item.isSelected && (
+                        <View
+                            style={[styles.flexRow, styles.alignItemsCenter]}
+                            accessible={false}
+                        >
+                            <View>
+                                <Icon
+                                    src={Expensicons.Checkmark}
+                                    fill={themeColors.success}
+                                />
+                            </View>
+                        </View>
+                    )}
+                </View>
+            </PressableWithFeedback>
+        );
+    };
+
+    useEffect(() => {
```

Comment:
It will be useful to leave a comment on what the purpose of that effect is, wdyt?

Reply:
Yep! Added here and in a few other places that were missing. Thanks!